### ISSUE TITLE
Disable colors in logs

### ISSUE THREAD

**Issue by pmichna** [tchiotludo/akhq#917]
It would be convenient to disable colors in logs. When storing logs in log management tools we can see unnecessary additional characters with color encoding instead of colors like:
```
2021-11-23 14:21:07,878 [34mINFO [0;39m [35mr-thread-7[0;39m [36morg.akhq.log.access       [0;39m [Date: 2021-11-23T14:21:07.878024Z] [Duration: 0 ms] [Url: GET /api/auths] [Status: 200] [Ip: /0.0.0.0] [User: xxx@example.com]
```

**Comment by radovanbabic** [comment]
I completely agree, it would be very convenient as it is causing problems and exceptions too. Maybe add option to switch the coloring off.

**Comment by tchiotludo** [comment]
PR are welcome on that, it's a simple Logback configuration, this one can be merge with
- [that discussion](https://github.com/tchiotludo/akhq/discussions/1060#discussioncomment-2501495) 
- https://github.com/tchiotludo/akhq/issues/91

**Comment by dorinand** [comment]
Is there any workaround to disable color for logs? Did you solve this somehow @pmichna ?

**Comment by pmichna** [comment]
@dorinand No, I didn't solve it. I just parse the non-JSON logs in our log management tool.

**Comment by ricantech** [comment]
Hi, what would be preferred solution?

enable/disable colors based on some property?
enable/disable json logs based on some property? 

Or prepare multiple built-in logback configurations and let people decide which one to use by specifying `logback.configurationFile`?

**Comment by dorinand** [comment]
Hi @ricantech, in my opinion best solution is to disable colors in logs. Additional characters are unnecessary and bring no value to logs. It also make it easier to parse log level from logs and process logs by harvesters like filebeat or promtail.

**Comment by ricantech** [comment]
I agree, the reason why I'm asking is to have some clear idea of whats expected. As from the discussion above and on other tickets I got the feeling it should be configurable.

I'm referring to comment done by @radovanbabic 

> I completely agree, it would be very convenient as it is causing problems and exceptions too. Maybe add option to switch the coloring off.

as @tchiotludo is owner of the repository he might have some opinion on how to tackle this?

**Comment by Gowtham-R-2002** [comment]
Hi @tchiotludo 👋

I'd like to take this issue — could you please assign it to me?

I've implemented a fix that allows switching the log pattern dynamically based on an environment variable

### Summary of the solution
- Added a new environment variable `LOG_TYPE` (`default` / `colored`) in `docker-compose.yml`.
- Updated `logback.xml` to select the appropriate log pattern dynamically using:
  ```xml
  <property name="activeEnv" value="${LOG_TYPE:-default}" />
  <property name="pattern" value="${pattern.${activeEnv}:-${pattern.default}}" />
Before proceeding with the PR, I’d like to confirm whether you’d prefer this environment variable (LOG_TYPE) to be defined directly within the docker-compose.yml file, or left for users to configure externally based on their setup.

**Comment by v27j11** [comment]
Hi @tchiotludo , can I get assigned on this ? I'll submit a PR for this within a week.
